src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -820,7 +823,8 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 }
                 else if (GetEmitter()->IsMovInstruction(ins))
                 {
-                    GetEmitter()->emitIns_Mov(ins, emitSize, targetReg, op1Reg, /* canSkip */ false, opt);
+                    bool canSkip = ((targetReg == op1Reg) && (intrin.baseType == intrin.op1->gtType));
```

Comment:
I wouldn't expect this one to ever be skippable. We'd have an integral base type and so the registers should never match up.